packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -676,6 +681,51 @@ export const announceVersions = ({
   })
 }
 
+/**
+ * announce updates
+ *
+ * @param updateStore
+ * @param capabilityStore
+ * @param configStore
+ * @param clientService
+ */
+export const announceUpdates = async ({
+  updatesStore,
+  capabilityStore,
+  configStore,
+  clientService
+}: {
+  updatesStore: UpdatesStore
+  capabilityStore: CapabilityStore
+  configStore: ConfigStore
+  clientService: ClientService
+}): Promise<void> => {
+  if (!capabilityStore.capabilities.core['check-for-updates']) {
+    return
+  }
+
+  try {
+    updatesStore.setIsLoading(true)
+    const { data }: { data: Updates } = await clientService.httpUnAuthenticated.get(
+      'https://update.opencloud.eu/server.json',
+      {
+        params: {
+          server: configStore.serverUrl,
+          edition: 'rolling', //TODO: retrieve serverEdition
```

Comment:
Could you already set the edition from the config here and do a fallback to `rolling`? Then it would automatically be used when the backend starts of include the edition.

Might be necessary to validate for a limited set of values (rolling or stable).